### DISCUSSION TITLE
found undefined alias in summary.yaml

### DISCUSSION THREAD

**Discussion by MarilenkusHA** [Madelena/hass-config-public#23] [Q&A]
Good day. I try to install your dashboards, but i have a problem: when i reload dashboard it have this erros

**found undefined alias 'summary-live-tile-swipe-card-style' in "/config/ui/views/summary.yaml", line 1115, column 24**

this line is

`style: *summary-live-tile-swipe-card-style`

How can i fix it?

**Comment by Madelena** [comment]
Alias is a way to reduce repetitive codes by telling the YAML loader to replace all `*name-of-the-alias` with what's inside `&name-of-the-alias`. So in this case it looks like you either had not copied the code for `&summary-live-tile-swipe-card-style` over, or had put the `*summary-live-tile-swipe-card-style` before `&summary-live-tile-swipe-card-style` in the file.
